common/Android.mk

```diff
@@ -83,6 +83,7 @@ endif
 LOCAL_CPPFLAGS += \
         -DHWC2_INCLUDE_STRINGIFICATION \
         -DHWC2_USE_CPP11 \
+        -DUSE_DRM_CTM \
```

Comment:
Should we have a BOARD Config to disable this ?
i.e ifneq ($(strip $(BOARD_DISABLE_DRM_CTM)), true)
LOCAL_CPPFLAGS += -DUSE_DRM_CTM
endif

Reply:
I see u revert this in next patch, instead let's have this board config, and selectively disable it on devices which show issues.